syntax = "proto3";
import "google/protobuf/empty.proto";

message NotificationRequest {
  string email_address = 1;
  string subject = 2;
  string content = 3;
}

service AlertSender {
  rpc SendNotification(NotificationRequest) returns (google.protobuf.Empty) {}
}
